### ISSUE TITLE
[BUG] empty WHITELIST_COUNTRY is ignored

### ISSUE THREAD

**Issue by lesensei** [bunkerity/bunkerweb#474]
**Description**
Not sure if this is a bug, an enhancement or a lack of doc that makes me miss something, but here goes: since I mainly use Bunkerweb to protect my home-hosted services, and have little need for foreign access to most of them, I setup the bunkerweb container to WHITELIST_COUNTRY only my own country. There's however one service which by essence can be accessed from anywhere (a Matrix Synapse server), and I can't "reset" this parameter for just that service, since it looks like an empty "WHITELIST_COUNTRY" is ignored (using autoconf with labels, by the way). Should it indeed reset the whitelist or should I simply stick with the workaround of setting the whitelist on all other individual services and not on the one where I don't need it ?

**How to reproduce**
docker-compose.yml for Bunkerweb:
```
services:
  bunkerweb:
    image: bunkerity/bunkerweb:1.5.0-beta
    environment:
      - AUTOCONF_MODE=yes
      - MULTISITE=yes
      [...]
      - WHITELIST_COUNTRY=FR
      - REVERSE_PROXY_WS=yes
    [...]
```

docker-compose.yml for Matrix:
```
services:
  synapse:
    image: matrixdotorg/synapse:latest
    [...]
    labels:
      bunkerweb.SERVER_NAME: "matrix.tldn"
      bunkerweb.USE_REVERSE_PROXY: "yes"
      bunkerweb.REVERSE_PROXY_URL: "/"
      bunkerweb.REVERSE_PROXY_HOST: "http://synapse:8008"
      bunkerweb.USE_CORS: "yes"
      bunkerweb.CORS_ALLOW_ORIGIN: "https://element.tldn"
      bunkerweb.CORS_ALLOW_HEADERS: Origin,X-Requested-With,Content-Type,Accept,Authorization
      bunkerweb.CORS_ALLOW_METHODS: "GET, POST, HEAD, OPTIONS, PUT"
      bunkerweb.ALLOWED_METHODS: GET|POST|HEAD|OPTIONS|PUT
      bunkerweb.REMOVE_HEADERS: Access-Control-Allow-Origin
      bunkerweb.WHITELIST_COUNTRY: ""
    [...]
```

I hope I didn't miss anything relevant in these extracts from my conf. Please tell me if I should provide anything else.

Result visible in UI is this for the Matrix service:
![image](https://github.com/bunkerity/bunkerweb/assets/1148387/7114020a-f8a5-4d8c-a066-30c141539387)

**Comment by TheophileDiot** [comment]
Hi, thank you for this issue, indeed a mistake has been made in a condition within the database, here's the commit that fixes it : https://github.com/bunkerity/bunkerweb/commit/c7ca5a822f471106fef87c56ff4bc7a727212050

This will be fixed in the next release.

**Comment by TheophileDiot** [comment]
Hello @lesensei 

This is fixed in the last [1.5.0 release](https://github.com/bunkerity/bunkerweb/tree/v1.5.0).

Thanks again !

**Comment by TheophileDiot** [comment]
Hi @lesensei it turns out this is only fix in the database and the UI but not on the backend :sweat_smile:

**Comment by fl0ppy-d1sk** [comment]
Hello @lesensei,

It should be fixed in the 1.5.1 release. Thanks for your report.
